src/helper.rs

```diff
@@ -71,8 +71,11 @@ impl<L: Loader + Send + Sync> HelperDef for FluentHelper<L> {
                 .filter_map(|(k, v)| {
                     let json = v.value();
                     let val = match *json {
-                        Json::Number(ref n) => FluentValue::Number(n.to_string()),
-                        Json::String(ref s) => FluentValue::String(s.to_string()),
+                        // `Number::as_f64` can't fail here because we haven't
+                        // enabled `arbitrary_precision` feature
+                        // in `serde_json`.
+                        Json::Number(ref n) => FluentValue::Number(n.as_f64().unwrap().into()),
+                        Json::String(ref s) => FluentValue::String(s.into()),
```

Comment:
You should be able to just `s.into()` and `n.into()`.

Reply:
It's slight more complicated as `n` is a `serde_json::Number` and `s` is an `&String`, you're right though that they could be simplified, I've updated the code now.